src/Compilers/Core/CodeAnalysisTest/Diagnostics/SuppressMessageAttributeCompilerTests.cs

```diff
@@ -35,6 +35,32 @@ private static Compilation CreateCompilation(string source, string language, str
             string fileName = language == LanguageNames.CSharp ? "Test.cs" : "Test.vb";
             string projectName = "TestProject";
 
+            const string unconditionalSuppressMessageDef = @"
+namespace System.Diagnostics.CodeAnalysis
+{
+    [System.AttributeUsage(System.AttributeTargets.All, AllowMultiple=true, Inherited=false)]
+    public sealed class UnconditionalSuppressMessageAttribute : System.Attribute
+    { 
+        public UnconditionalSuppressMessageAttribute(string category, string checkId)
+        {
+            Category = category;
+            CheckId = checkId;
+        }
+        public string Category { get; }
+        public string CheckId { get; }
+        public string Scope { get; set; }
+        public string Target { get; set; }
+        public string MessageId { get; set; }
+        public string Justification { get; set; }
+    }
+}";
+            var mscorlibRef = new[] { TestBase.MscorlibRef };
+
+            var refComp = CSharpCompilation.Create("unconditionalsuppress",
+                options: new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary),
+                syntaxTrees: new[] { CSharpSyntaxTree.ParseText(unconditionalSuppressMessageDef) },
+                references: mscorlibRef).EmitToImageReference();
```

Comment:
>mscorlibRef [](start = 28, length = 11)

Taking dependency on particular version of core library feels fragile.

Reply:
I see that we are already doing this below. Consider using this local in all places to make sure consistency across all compilations.

---
In reply to: [595318758](https://github.com/dotnet/roslyn/pull/51792#discussion_r595318758) [](ancestors = 595318758)